app/src/main/java/fr/free/nrw/commons/upload/worker/UploadWorker.kt

```diff
@@ -180,12 +182,16 @@ class UploadWorker(var appContext: Context, workerParams: WorkerParameters) :
                         if (contribution.state == Contribution.STATE_QUEUED) {
                             contribution.state = Contribution.STATE_QUEUED_LIMITED_CONNECTION_MODE
                             contributionDao.save(contribution)
+                        } else {
+                            //To Nothing
```

Comment:
Are these two lines really needed?

Reply:
Yes.Else statement fix :'if' must have both main and 'else' branches if used as an expression